app/lib/entity_cache.rb

```diff
@@ -8,7 +8,7 @@ class EntityCache
   MAX_EXPIRATION = 7.days.freeze
 
   def mention(username, domain)
-    Rails.cache.fetch(to_key(:mention, username, domain), expires_in: MAX_EXPIRATION) { Account.select(:username, :domain, :url).find_remote(username, domain) }
+    Rails.cache.fetch(to_key(:mention, username, domain), expires_in: MAX_EXPIRATION) { Account.select(:id, :username, :domain, :url).find_remote(username, domain) }
```

Comment:
Why this change?
Actually, I'm not sure what this line does at all, since `AccountFinderConcern#find_remote` seems to perform a search with its own scope?

Reply:
Without this the retrieved model doesn't have an `id` attribute. Scopes are composable.